README.md

```diff
@@ -1,2 +1,41 @@
-# civis-integrations-sdk-security-orb
-A CircleCI orb with some common security checks for Python repositories.
+# Orb Project Template
```

Comment:
I looked this over and made a couple of tweaks. A lot of these instructions should stay there, there's a specific process where CircleCI will publish new versions of an orb based on the tag in the PR.

I also tested this release process a few days ago and it created a [1.0.0 release](https://circleci.com/developer/orbs/orb/civisanalytics/civis-integrations-sdk-security-orb). I changed the title of this PR to patch so the next release will be 1.0.1 instead of 2.0.0, that's why the title is different now.